tests/unit/test_parsers.py

```diff
@@ -547,7 +547,7 @@ def test_parsers(parser_class, raw_file, results_file):
     assert parsed_notifications == expected_result
 
 
-@pytest.mark.parametrize("parser_class", [ICal, EmailDateParser, HtmlParserZayo1, SubjectParserSeaborn1])
+@pytest.mark.parametrize("parser_class", [ICal, EmailDateParser, HtmlParserZayo1, SubjectParserZayo1])
```

Comment:
Should this be an addition to the test rather than replacing `SubjectParserSeaborn1`?

Reply:
Should it be Seaborn1? 
This test case parameterises a parser class with Zayo HTML parser and Seaborn1 Subject parser.

Reply:
Having said that, I'm more than happy to fix it up with an additional test case if `
@pytest.mark.parametrize("parser_class", [ICal, EmailDateParser, HtmlParserZayo1, SubjectParserSeaborn1])` in intentional.

Reply:
Yes, it is more consistent with the test to use the same Parser type, as a side fix in the PR.